lms/djangoapps/instructor/views/api.py

```diff
@@ -2910,3 +2910,98 @@ def generate_certificate_exceptions(request, course_id, generate_for=None):
     }
 
     return JsonResponse(response_payload)
+
+
+USER_INDEX = 0
```

Comment:
Since we are using these constants only once in `generate_bulk_certificate_exceptions` view i think we should make them local.